atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableDefinition.java

```diff
@@ -343,7 +351,7 @@ public TableMetadata toTableMetadata() {
                 sweepStrategy,
                 expirationStrategy,
                 appendHeavyAndReadLight,
-                tableNameLoggable);
+                tableNameSafety.isSafeForLogging());
```

Comment:
Are we unwrapping the enum here to avoid breaking the previous PR?

I see that we released 0.49.0 with `TableMetadata` and `NameComponentDescription` having those boolean args. Even so, is anyone even using those args yet?